Update README.md (#2642)
ipfs/ipfs-desktop: README.md

npm start
```

> > **Build Note:** `npm ci` will download the webui code to run in electron from the ipfs network using the bitswap protocol.  For details, see the [build process](`.github/workflows/ci.yml`) and the [webui code](https://github.com/ipfs/ipfs-webui).

Reviewer: Somewhat correct except we use ipfs-or-gateway npm package

```suggestion
> **Build Note:** `npm ci` will download the webui code to run in electron from IPFS using the [ipfs-or-gateway](https://www.npmjs.com/package/ipfs-or-gateway) npm package.  For details, see the [build process](`.github/workflows/ci.yml`) and the [webui code](https://github.com/ipfs/ipfs-webui).
```

> ipfs-desktop in itself is a simple container that makes sure kubo and webui can work together in a standalone fashion and has access to other os-specfic features like tray and contextual integrations.

Reviewer: ```suggestion
IPFS Desktop in itself is a simple container that makes sure Kubo and IPFS Webui can work together in a standalone fashion and has access to other os-specfic features like tray and contextual integrations.
```

> There are multiple ways to access webui:

Reviewer: ```suggestion
There are multiple ways to access IPFS Webui:
```

 
- https://webui.ipfs.io/#/welcome
- http://127.0.0.1:5001/webui (shipped with kubo)

> - ipfs-desktop itself

Reviewer: ```suggestion
- IPFS Desktop itself
```

> All of these instances of webui are the same but shipped slightly differently. The file you're seeing being downloaded is the latest release of webui, i.e. v4.1.1 has content identifiers (CID) `bafybeiamycmd52xvg6k3nzr6z3n33de6a2teyhquhj4kspdtnvetnkrfim` which can be verified on the [webui release page](https://github.com/ipfs/ipfs-webui/releases).

Reviewer: ```suggestion
All of these instances of IPFS Webui are the same but shipped slightly differently. The file you're seeing being downloaded is a specific release of IPFS Webui, i.e. v4.1.1 has content identifiers (CID) `bafybeiamycmd52xvg6k3nzr6z3n33de6a2teyhquhj4kspdtnvetnkrfim` which can be verified on the [IPFS Webui release page](https://github.com/ipfs/ipfs-webui/releases).
```
